src/node_context_data.h

```diff
@@ -21,26 +21,62 @@ namespace node {
 #define NODE_CONTEXT_ALLOW_WASM_CODE_GENERATION_INDEX 34
 #endif
 
-#ifndef NODE_CONTEXT_TAG
-#define NODE_CONTEXT_TAG 35
-#endif
-
 #ifndef NODE_BINDING_LIST
-#define NODE_BINDING_LIST_INDEX 36
+#define NODE_BINDING_LIST_INDEX 35
 #endif
 
 #ifndef NODE_CONTEXT_ALLOW_CODE_GENERATION_FROM_STRINGS_INDEX
-#define NODE_CONTEXT_ALLOW_CODE_GENERATION_FROM_STRINGS_INDEX 37
+#define NODE_CONTEXT_ALLOW_CODE_GENERATION_FROM_STRINGS_INDEX 36
+#endif
+
+// NODE_CONTEXT_TAG must be greater than any embedder indexes so that a single
+// check on the number of embedder data fields can assure the presence of all
+// embedder indexes.
+#ifndef NODE_CONTEXT_TAG
+#define NODE_CONTEXT_TAG 37
 #endif
 
 enum ContextEmbedderIndex {
   kEnvironment = NODE_CONTEXT_EMBEDDER_DATA_INDEX,
   kSandboxObject = NODE_CONTEXT_SANDBOX_OBJECT_INDEX,
   kAllowWasmCodeGeneration = NODE_CONTEXT_ALLOW_WASM_CODE_GENERATION_INDEX,
-  kContextTag = NODE_CONTEXT_TAG,
   kBindingListIndex = NODE_BINDING_LIST_INDEX,
   kAllowCodeGenerationFromStrings =
-      NODE_CONTEXT_ALLOW_CODE_GENERATION_FROM_STRINGS_INDEX
+      NODE_CONTEXT_ALLOW_CODE_GENERATION_FROM_STRINGS_INDEX,
+  kContextTag = NODE_CONTEXT_TAG,
+};
+
+class ContextEmbedderTag {
+ public:
+  static inline void TagNodeContext(v8::Local<v8::Context> context) {
```

Comment:
This header now uses V8 APIs, so it should ideally `#include` the relevant V8 headers as well.

Reply:
Thanks for pointing this out. Updated :)